bcda/models/models.go

```diff
@@ -270,6 +272,32 @@ func AssignAlphaBeneficiaries(db *gorm.DB, aco ACO, acoSize string) error {
 	return db.Exec(s).Error
 }
 
+type Group struct {
+	ID   string         `gorm:"primary_key" json:"id"`
+	Data postgres.Jsonb `json:"data"`
```

Comment:
I believe it is an option to use a string here, since we are validating the structure of the json before it goes in and when it comes out simply by marshaling it in/out of structs and are not planning on using postgres's jsonb query capabilities.